main.go

```diff
@@ -10,8 +10,27 @@ import (
 
 	"github.com/shawn-hurley/jsonrpc-golang/jsonrpc2"
 	"github.com/shawn-hurley/jsonrpc-golang/lsp/protocol"
+	"github.com/shawn-hurley/jsonrpc-golang/rules"
 )
 
+//TODO(shawn-hurley) - this needs to be passed in as args. Will need to refactor to use cobra
+var config = rules.Configuration{
+	ProjectLocation: "./examples/golang",
+}
+
+//TODO(shawn-hurley) - this package/type name stutters.
+var workspaceRules = []rules.Rule{
```

Comment:
rules Rule sounds like a 90s ad campaign

Reply:
Yeah, this name is bad, reason for the TODO